gmt/modules.py

```diff
@@ -0,0 +1,31 @@
+"""
+Non-plot GMT modules.
+"""
+from tempfile import NamedTemporaryFile
+
+from .clib import LibGMT
+from .utils import build_arg_string
+from .decorators import fmt_docstring, use_alias, kwargs_to_strings
+
+
+@fmt_docstring
+def info(data, **kwargs):
+    """
+    Get information about data tables.
+
+    {gmt_module_docs}
+
+    Parameters
+    ----------
+    data: str
+        A data file name.
```

Comment:
Let's keep it only for data files for now. We can think about adding pandas.Dataframe later but I don't know if it will be helpful. 

In this case, `fname` is a better name for the argument. I'll probably refactor `plot` to use `fname` for files as well.